schema.hh

```diff
@@ -641,6 +642,7 @@ private:
         // we will use by default - when we have the choice.
         sstables::compaction_strategy_type _compaction_strategy = sstables::compaction_strategy_type::size_tiered;
         std::map<sstring, sstring> _compaction_strategy_options;
+        tombstone_gc_options _tombstone_gc_options;
```

Comment:
This needs may to participate in hashing. Need to make sure it propagates correctly to other nodes.

Reply:
I think schema hashing is done on schema mutations. So store_map() is sufficient.